### PR TITLE
Add new CRTM version v2.4_jedi

### LINKED ISSUE #155: requesting updates to package.py for CRTM
@climbfuji 

https://github.com/NOAA-EMC/spack/blob/8a949c880aac1781852eb8c1ef45f32668edcdfb/var/spack/repos/builtin/packages/crtm/package.py

Here are the relevant tags for CRTM (https://github.com/JCSDA/crtm):
For JEDI use:
`v2.4_jedi`     ( tag newly created, points to `release/crtm_jedi_v2.4.0`)
`v2.3_jedi.4`  ( points to `release/crtm_jedi`  last digit incremented with updates to the release branch) 

For EMC use:
`v2.4.0_emc`   ( tag newly created, points to `release/REL-2.4.0_emc` )
`v2.3.0_emc`   ( tag newly created, points to `release/REL-2.3.0_emc` ) 

EMC releases use `git-lfs` for the binary `fix/` directory, but an alternative exists for systems with no `git-lfs` support.

jedi releases rely on `ctest` to download fix/ coefficients from GDEX.    This download should probably occur during the build process, rather than the ctest. 

This is all that `spack-stack` needs to support right now.   The  "non-EMC" and "non-jedi" releases of CRTM do not need to be in `spack stack`.

### PR DESCRIPTION
**Pull request by climbfuji**
As the title says - also update the list of maintainers to more accurately reflect who is maintaining the CRTM package in spack, and fix the URL to point to the authoritative CRTM repository.

See https://github.com/NOAA-EMC/spack/issues/155 for a detailed description.

Fixes #155

### CODE REVIEW

**Comment by climbfuji** [comment]
@BenjaminTJohnson FYI - please take a look and let me know what you think. Thanks!

File: var/spack/repos/builtin/packages/crtm/package.py [hunk 1]
```diff
@@ -41,5 +41,8 @@ class Crtm(CMakePackage):
     version("2.4.0", commit="a831626")
     # Uses the tip of REL-2.3.0_emc branch
     version("2.3.0", commit="99760e6")
```
Comment (BenjaminTJohnson) [inline var/spack/repos/builtin/packages/crtm/package.py:45]: Assuming these version values are tags or should be consistent with tags, this should be "v2.4.0_emc" and "v2.3.0_emc"

`v2.4.0_emc` tag has the commit: "5ddd0d6"
`v2.3.0_emc` tag has the commit: "99760e6"

`v2.3-jedi.4` is correct

`v2.4_jedi` should be "0ee3593"

Not sure why there's a discrepancy between internal and public repos on the commit hashes for the tags. I'll investigate, but for now, let's use what's on public only.

Let me know if I'm misunderstanding something.

**Review by BenjaminTJohnson — CHANGES_REQUESTED** [review]

**Comment by climbfuji** [comment]
Thanks, you are right and I was wrong. Will make those changes.

**Review by srherbener — APPROVED** [review]

**Comment by climbfuji** [comment]
I am building all four versions of CRTM on my macOS now, and I found a bug or two - will update the PR in a bit.

**Comment by climbfuji** [comment]
@srherbener @BenjaminTJohnson I was able to install all four versions of CRTM on my macOS after the latest bug fixes that I just pushed.

**Comment by climbfuji** [comment]
Since @BenjaminTJohnson is on leave for three weeks and I used his corrected hashes, and since the PR was approved and everything built as expected, I am going to merge this once the unit tests passed.

**Pull request merged**